_docs/_latest/configure/gsuite.md

```diff
@@ -0,0 +1,63 @@
+---
+title: G Suite
+order: 600
+---
+
+#  {{ page.title }}
+
+This page describes how to enable the data collection of G Suite Google Groups for
```

Comment:
We collect more than groups. Can remove any references to groups here and elsewhere in this PR?